app/src/main/java/org/feature/fox/coffee_counter/data/repository/UserRepository.kt

```diff
@@ -58,6 +71,10 @@ class UserRepository @Inject constructor(
         return userDao.getPurchaseListOfUser(id)
     }
 
+    override suspend fun getImageByIdFromUser(id: String): Image? {
+        return userDao.getImageById(id)
+    }
+
```

Comment:
Pls add Db at the end of the method name to stay consistent and provide better Overview